src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,7 +42,8 @@ public RobotContainer() {
             () -> -driverController.getLeftY(),
             () -> -driverController.getRightX(),
             () -> driverController.getRightTriggerAxis() > .1,
-            drivetrain));
+            drivetrain,
+            indications.getCurrentStateTopic()));
```

Comment:
I don't really like this. It might be better to make `Indications#getCurrentStateTopic()` static to avoid passing something in constructors like this.